### DISCUSSION TITLE
Why VerifyFunc in DatePicker not Work?

### DISCUSSION THREAD

**Discussion by MasterBran** [HandyOrg/HandyControl#1161] [Q&A] [answered]
i write this in .xaml file
`<hc:DatePicker 
                        Width="360" 
                        Name="StartDate"
                        hc:InfoElement.TitleWidth="64" 
                        Style="{StaticResource DatePickerExtend}"
                        hc:InfoElement.TitlePlacement="Left"
                        hc:InfoElement.Placeholder="2019-10-28"
                        hc:InfoElement.Title="开始日期" 
                        hc:InfoElement.Necessary="True"
                        hc:InfoElement.Symbol="*"/>`
and this in .cs file
` StartDate.VerifyFunc = str =>
            {
                return OperationResult.Failed("cant be empty");
            };`
but not work

**Comment by ghost1372** [comment, marked as answer]
VerifyFunc is removed in next version.
Wait for new version with validation support
